validation/custom_constraint.rst

```diff
@@ -178,15 +178,15 @@ with the necessary ``validator.constraint_validator``. This means you can
 Class Constraint Validator
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Beside validating a class property, a constraint can have a class scope by
-providing a target in its ``Constraint`` class::
+Besides validating a single property, a constraint can have an entire class
+as its scope. Just add this to the ``Constraint`` class::
```

Comment:
```suggestion
as its scope. You only need to add this to the ``Constraint`` class::
```

Reply:
:-)

Reply:
I made this change while merging. It's important to not use words like *"Just"* in this way because lots of newcomers consider them "nocive". We do this because nothing is "simple", "easy", "obvious" or "just ..." when you are starting to learn something.

Reply:
@javiereguiluz: I see your point. But, to be honest, I can't see much of a difference between "just" and "only" ;-)

Anyway, off-topic question: When exactly are changes going live at https://symfony.com/ ? Is there a "release" process? Or just some delay?

Reply:
I don't know the exact details ... but docs are built several times per day, so updates are usually quick. Thanks.